web/src/test/java/se/inera/intyg/webcert/web/service/underskrift/dss/DssSignMessageIdpProviderTest.java

```diff
@@ -123,4 +126,30 @@ void shallUseDefaultIdpIfIdentityProviderForSignIsEmpty() {
             );
         }
     }
+
+    @Nested
+    class UseSameDeviceWhenMtlsTest {
+
+        @BeforeEach
+        void setUp() {
+            dssSignMessageIdpProvider = new DssSignMessageIdpProvider(
+                DEFAULT_IDP_URL,
+                true,
+                "/saml/sign");
+        }
+
+        @Test
+        void shallUseDefaultIdpIfIdentityProviderForSignIsMtls() {
+            assertEquals(DEFAULT_IDP_URL,
+                dssSignMessageIdpProvider.get(IDENTITY_PROVIDER_FOR_SIGN_WHEN_MTLS)
+            );
+        }
+
+        @Test
+        void shallUseIdentityProviderForSignIfNotMtls() {
+            assertEquals(DEFAULT_IDP_URL,
+                dssSignMessageIdpProvider.get(IDENTITY_PROVIDER_FOR_SIGN_WHEN_MTLS)
+            );
+        }
```

Comment:
The two tests above appear to be identical?

Reply:
Thanks! I didn't complete the test as planned! Nice catch! :-)

Reply:
Thats more like it